CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+If you wish to propose changes to the extension list, you can create a branch, edit the list within the "extensionPack" section of the package.json file, and create a pull request. If you wish to change the major version of the extension pack, this can be done by manually editing the "version" section of the package.json. Minor and patch versions are incremented automatically when built, minor if a "master" branch build, otherwise patch.
```

Comment:
I think we should have some guidance as to what would consider a major (meaning breaking) change. I presume it could be:
* removal of an extension
* incrementing the minimum VS Engine version? (although I suspect this would be difficult as would always inherit max VS engine version from all dependencies).

Does VSCode have any guidance on applying semver?

Reply:
I've added the requested guidance. In terms of vscode guidance, I can only really see the following - 
![image](https://github.com/Aqovia/VisualStudioCode.ExtensionPack/assets/138611636/6d31285a-b25f-4699-b600-6aaaec7f656b)